docs/git-workflow.md

```diff
@@ -1,36 +1,67 @@
 # Git branching workflow for narrative
 
-- Based on: http://drewfradette.ca/a-simpler-successful-git-branching-model/
-- Last modified: Dec. 11, 2014
+This document describes the git workflow that *all- developers of the narrative code must follow. It also describes the release procedure, which only an unlucky few should have to worry about.
 
-This document describes the git workflow that *all* developers of the narrative code must follow. It also describes the release procedure, which only an unlucky few should have to worry about.
+- Last modified: Aug. 31, 2020
+
+***Table of Contents***
+
+- [Branches and Tags](#branches-tags)
+- [Contributing](#Contributing)
+  - [Pull Requests](#pull-requests)
+- [Release Flows](#realease-flow)
+  - [Development Releases](#development-releases)
+  - [Production Releases](#production-releases)
+  - [Hotfix Releases](#hotfixes-releases)
 
 ## Branches and Tags
 
 There are 3 branches:
 
-* `develop` – Development branch. New features are added off this branch.
-* `staging` – Staging or "release candidate" branch. The branch deployed `narrative-dev`.
-* `master` – Production branch. The branch deployed on `narrative-next` and `narrative`. This branch is used for hotfixes.
+- `develop` – Development branch. New features are added off this branch.
+- `staging` – Staging or "release candidate" branch. The branch deployed `narrative-dev`.
+- `master` – Production branch. The branch deployed on `narrative-next` and `narrative`. This branch is used for hotfixes.
 
 Tags on master are used for indicating releases (see "Production releases", below).
 
-## Workflow
+## Contributing
 
-There are three main workflows: development, production releases, and hotfixes.
+To contribute to this repository create a pull request for a new feature or issue.
+
+### Pull Requests
+
+#### _Internal Developers_
+
+To begin working on a feature or story, make sure you've assigned yourself to the story/issue in JIRA and marked it as "In Progress". If there isn't an story/issue yet for what you want to work on, please create one.
+
+Create a new branch off `develop` using the naming convention:
+
+`{your initials or username}-{summary}-{JIRA #}`
```

Comment:
is this a new thing or an existing thing? If it's a new thing, I think it'd be better to have the JIRA namespace included, and reorder the parts of the branch name thus:

`{JIRA #}-{summary}-{user ID}`

e.g.

`DATAUP-63-data_panel_button-jd`

I'm not even sure if the user ID is all that important, esp. since it's likely to get confused with the other text; what happens if there are several people collaborating on the same branch? The most informative version might be:

`{JIRA #}-{summary}`
e.g. `DATAUP-63-data_panel_button`

Reply:
That's fine with me! I like the JIRA then summary. Having a naming convention just helps tame the chaos of lots of branches :)